package.json

```diff
@@ -6,6 +6,7 @@
   "license": "Apache-2.0",
   "private": true,
   "scripts": {
+    "start": "ts-node packages/issuer/lib/AuthServer.ts",
```

Comment:
Please move that to the actual module. The REST API for instance will move to another module. So make it the concern of these modules themselves

Reply:
Fixed!